docker-compose-single.yml

```diff
@@ -1,8 +1,10 @@
-version: "3.3"
+version: "3.8"
 services:
   iroha0:
     build: .
     image: iroha0:debug
+    volumes:
+      - './:/root/soramitsu/iroha'
```

Comment:
The local repository path varies by developer though.
What is the reason we share it?

Reply:
Yes, they vary. That is why `.`(current dir) is mapped to `/root/soramitsu/iroha` in container. It doesn't need to be mapped, I needed a way to copy locally built `iroha` into docker container and run it. I can get away with `docker cp` but maybe someone would like to build `iroha` inside container and now they have the possibility with shared volume

Reply:
I mistook right for left of `:`